### ISSUE TITLE
Файл modal.scss

### ISSUE THREAD

**Issue by Morfey216** [MrRobot-p/Meine-coon#7]
1. Нарушение БЭМ !!!
Что делает в файле `modal.scss` код стилей других блоков? https://github.com/MrRobot-p/Meine-coon/blob/534eb7b2243742b29a9cb949b18d8a5fcb43a357/src/scss/components/modal/modal.scss#L182-L216
2. У флекс-боксов `flex-direction: row;` указано по умолчанию, если ранее направление оси не переопределялось, то прописывать значение не надо https://github.com/MrRobot-p/Meine-coon/blob/534eb7b2243742b29a9cb949b18d8a5fcb43a357/src/scss/components/modal/modal.scss#L100-L101 https://github.com/MrRobot-p/Meine-coon/blob/534eb7b2243742b29a9cb949b18d8a5fcb43a357/src/scss/components/modal/modal.scss#L157-L158

**Comment by MrRobot-p** [comment]
Исправлено

**Comment by Morfey216** [comment]
Нет, первый пункт не исправлен... Ниже приведены блоки, которые, согласно БЭМ, не имеют отношения к блоку `modal`
https://github.com/MrRobot-p/Meine-coon/blob/ee10dd3b3572c47d6854e99ea716f7b67990d370/src/scss/components/modal/modal.scss#L17
https://github.com/MrRobot-p/Meine-coon/blob/ee10dd3b3572c47d6854e99ea716f7b67990d370/src/scss/components/modal/modal.scss#L21
https://github.com/MrRobot-p/Meine-coon/blob/ee10dd3b3572c47d6854e99ea716f7b67990d370/src/scss/components/modal/modal.scss#L26
https://github.com/MrRobot-p/Meine-coon/blob/ee10dd3b3572c47d6854e99ea716f7b67990d370/src/scss/components/modal/modal.scss#L43-L44
https://github.com/MrRobot-p/Meine-coon/blob/ee10dd3b3572c47d6854e99ea716f7b67990d370/src/scss/components/modal/modal.scss#L55
https://github.com/MrRobot-p/Meine-coon/blob/ee10dd3b3572c47d6854e99ea716f7b67990d370/src/scss/components/modal/modal.scss#L59

**Comment by MrRobot-p** [comment]
Имеется ввиду - (Неверно: .modal-show Верно: .modal__modal-show)?

**Comment by Morfey216** [comment]
Тут такой момент: исходя из названия классов не понятно - это другой блок или просто ошибка в названии. Если ошибка - достаточно просто исправить на новый вариант. Если же это действительно другой блок, то его необходимо вынести в собственный файл стилей

**Comment by MrRobot-p** [comment]
Переосмыслил, поправил как нужно
